times_reader/transforms.py

```diff
@@ -448,9 +448,6 @@ def process_user_constraint_table(
         # Fill in UC_N blank cells with value from above
         df["UC_N"] = df["UC_N"].ffill()
 
-        if "UC_Desc" in df.columns:
-            df.drop("UC_Desc", axis=1, inplace=True)
-
```

Comment:
Why was this removed, btw?

Reply:
Because we now need that column for the UC_N mapping: